src/constants/authForms.js

```diff
@@ -0,0 +1,100 @@
+export const AUTH_FORM_KEYS = {
+  LOGIN: 'login',
+  REGISTER: 'register',
+  VERIFY_EMAIL: 'verify-email',
+  FORGOT_PASSWORD: 'forgot-password',
+  SET_PASSWORD: 'set-password',
+};
+
+const authForms = {
```

Comment:
It's good to separate the layout from the data that's filling the layout. Good job.